docs/CHANGELOG.md

```diff
@@ -0,0 +1,33 @@
+<!--template:
+# unreleased - YYYY-MM-DD
+## Python API
+
+## CLI
+
+## Catalog
+!-->
+
+# unreleased - YYYY-MM-DD
+## Python API
+* Improve performance of hashing of S3 files (#1816)
+
+## CLI
+
+## Catalog
+
+# 3.2.0 - 2020-09-08 - Package Registry Refactor
+## Python:
+* Refactors local and s3 storage-layer code around a new PackageRegistry base class (to support improved file layouts in future releases)
+* Multithreaded download for large files, large performance gains when installing packages with large files, especially on large instances
+* Package name added to Package.resolve_hash
+* Bugfix: remove package revision by shorthash
+* Performance improvements for build and push
+
+## Catalog & Lambdas:
+* PDF previews
+* Browse full package contents (no longer limited to 1000 files)
+* Indexing and search package-level metadata
+* Fixed issue with download button for certain text files
+* FCS files: content indexing and preview
+* Catalog sign-in with email (or username)
+* Catalog support for sign-in with Okta
```

Comment:
These are the notes from the latest [release](https://github.com/quiltdata/quilt/releases/tag/v3.2.0).
If we want, we could populate it from all previous releases doing this:
```bash
pip install PyGithub
```
```python
from github import Github
g = Github()
repo = g.get_repo('quiltdata/quilt')
releases = [r for r in r.get_releases() if not r.prerelease]
data = '\n\n'.join([
    f'# {r.tag_name.lstrip("v")} - {r.published_at.date()} - {r.title}' + (f'\n{r.body.rstrip()}' if r.body else '')
    for r in releases
])
with open('docs/CHANGELOG.md', 'w') as f:
    f.write(data)
```